warcraft/contracts/assets/stateful.py

```diff
@@ -0,0 +1,62 @@
+from pyteal import *
+
+def approval_program():
+    app_manager = Bytes("app_manager")
+    total_registered = Bytes("total_registered")
+    escrow = Bytes("escrow")
```

Comment:
Can we rename this to `escrow_addr`?

Reply:
i think escrow is enough, addr seems extra.

Reply:
wasn't sure what `escrow` represents upon first look. had to check the code.